src/transformers/diffLineTransformer.js

```diff
@@ -0,0 +1,47 @@
+// @ts-check
+const { highlightDiffLine } = require('./transformerUtils');
+
+function createDiffLineTransformer() {
+  // How do I get the parsed code fence options here?
+  /** @type {LineTransformer<any>} */
+  const diffLineTransformer = ({ line, meta: { diff } }) => {
+    if (diff && line.text.startsWith('+ ')) {
```

Comment:
GitHub’s diff grammar doesn’t require the space... do you think we would get false positives if we didn’t require it? I can’t think of many times where an actual line of source code begins with a `+` (without any leading indentation), but surely some rare cases exist. I wonder if it would also be worth it to allow escapes with `\+`... probably not, but I think we should consider dropping the space.

Reply:
Should we still strip off a space after the `+/-` (if there is one)? Or just leave it as is?

```js
const text = line.text.replace(/^\+/, '');   // or
const text = line.text.replace(/^\+ ?/, '');
return {
  line: highlightDiffLine({ ...line, text }, 'add'),
```

Reply:
If a space isn’t necessary to trigger the highlight, then a space should not be stripped.

Reply:
Wait, on second thought, this seems fine. One space can get stripped. This is a lot like Markdown’s treatment of the optional space after block quote markers (`>`).

Reply:
> This is a lot like Markdown’s treatment of the optional space after block quote markers (`>`).

Exactly, that's what I was thinking. Not checking for a space in the `if(diff && line.text.startsWith('-'))` statement anymore but still stripping one (if available) now.